### ISSUE TITLE
Assembly merged into one giant scaffold

### ISSUE THREAD

**Issue by robertriley6** [aidenlab/3d-dna#120]
Hi,

I ran 3d-dna on a fungal genome as

run-asm-pipeline.sh references/assembly.fasta aligned/merged_nodups.txt

Previously I had run juicer as 

juicer.sh -d $PWD -z references/assembly.fasta -y restriction_sites/nyp_Arima.txt -p references/name.length

and it failed, but generated data I needed in aligned/ to run 3d-dna.  Which I then ran, and it generated an assembly with one huge scaffold (basically the whole genome).  N50 of 4 is now an N50 of 1, which I don't think is realistic.

Is there some parameter I can tweak to make 3d-dna more conservative about scaffold joins?  The genome seems to be repetitive, and I wonder if that's an issue, e.g. Hi-C reads mapping to many repeat instances in the genome.

Thanks

![image](https://user-images.githubusercontent.com/30445518/120723217-2e840d80-c486-11eb-82b4-a80dd470b0f8.png)

**Comment by dudcha** [comment]
Hello,

The standard recommendation is to examine the .hic files to help understand the issue. Between the .out and .err and .0.hic -> .rawchrom.hic files you should be able to see if there is meaningful signal to begin with and if so, which part of the pipeline is causing you trouble.

Hope this helps,
Olga

**Comment by wen10-git** [comment]
Hello, @dudcha are you online recently?

I may have encountered a similar problem. According to my situation and my point of view, maybe the chromosome boundaries generated by the rawchrome file are more reliable. So I chose the result produced by the rawchrome file. But I'm still a bit confused as to why the .0 files are behaving like that. 

My running command is roughly as follows: 
run-asm-pipeline.sh -r 0 reference/Rb.genome.fa aligned/merged_nodups.txt &> 3d.log & 

and it still can't finish running, Stuck at the end.

0.hic and 0.assembly heat maps generated via juicerbox. 



And, heatmaps generated by rawchrom.hic and rawchrom.assembly via juicerbox.


Hello, @dudcha are you online recently?

I may have encountered a similar problem. According to my situation and my point of view, maybe the chromosome boundaries generated by the rawchrome file are more reliable. So I chose the result produced by the rawchrome file. But I'm still a bit confused as to why the .0 files are behaving like that. 

My running command is roughly as follows: 
run-asm-pipeline.sh -r 0 reference/Rb.genome.fa aligned/merged_nodups.txt &> 3d.log & 

and it still can't finish running, Stuck at the end.

0.hic and 0.assembly heat maps generated via juicerbox. 

![屏幕截图 2023-05-20 235901](https://github.com/aidenlab/3d-dna/assets/63379795/a4c67f60-ec56-455a-a152-1569863afdd2)


And, heatmaps generated by rawchrom.hic and rawchrom.assembly via juicerbox.


![屏幕截图 2023-05-21 005016](https://github.com/aidenlab/3d-dna/assets/63379795/a54ad995-26b5-40c7-907b-97fefb308ef5)
